Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.8.6-slim-buster AS production
```

Comment:
These two stages are almost entirely duplicated - we can simplify our setup by sharing as many of these stages as we can, e.g.
```
FROM python:3.8.6-slim-buster AS base
RUN apt-get ...
<all shared stages>

FROM base AS production
<prod only config>

FROM base AS development
<dev only config>
```